contracts/colony/Colony.sol

```diff
@@ -20,10 +20,12 @@ pragma experimental ABIEncoderV2;
 
 import "./../common/ERC20Extended.sol";
 import "./../common/IEtherRouter.sol";
+import "./../common/MultiChain.sol";
+import "./../extensions/ColonyExtension.sol";
```

Comment:
Is this doing anything?

Reply:
`ColonyExtension` is used in the file... I guess it's able to pick it up from the import in ColonyStorage?

Reply:
Yeah, it doesn't need this to compile successfully.

Reply:
I mention it because the PR doesn't introduce it anywhere other than the import, which tells me that any existing usage is already functioning.